<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="USE2DEVP01047" failures="1" tests="5" name="Test" time="95.188" errors="0" timestamp="2024-02-27T16:39:19 IST">
  <testcase classname="RahulSheetyAcademy.MavenProjectUdemy.errroValidationTest" name="loginErrorValidation" time="1.578"/>
  <testcase classname="RahulSheetyAcademy.MavenProjectUdemy.errroValidationTest" name="productErrorValidation" time="14.901"/>
  <testcase classname="RahulSheetyAcademy.MavenProjectUdemy.submitOrderTest" name="submitOrder" time="16.27"/>
  <testcase classname="RahulSheetyAcademy.MavenProjectUdemy.submitOrderTest" name="submitOrder" time="14.71">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RahulSheetyAcademy.MavenProjectUdemy.submitOrderTest.submitOrder(submitOrderTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- submitOrder -->
  <testcase classname="RahulSheetyAcademy.MavenProjectUdemy.submitOrderTest" name="orderHIstory" time="0.0">
    <skipped/>
  </testcase> <!-- orderHIstory -->
</testsuite> <!-- Test -->
